### PR TITLE
Prepare 0.0.9

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

**Comment by ralphtheninja** [comment]
TODO: Update README with information from https://github.com/airtap/airtap/pull/194

File: doc/travis-ci.md [hunk 1]
```diff
@@ -43,7 +43,39 @@ That's it! Lets look at the important lines.
...
+
+```json
+{
+  "scripts": {
+    "test": "airtap --sauce-connect --loopback airtap.local --coverage test.js"
+  }
+}
+```
+
+Additionally, you can add hook in Travis to post the results to `coveralls`:
```
Comment (vweevers) [inline doc/travis-ci.md]: "add hook in Travis" => perhaps just "configure Travis".

I'm not sure I was right to call this a "hook" in #194, might be "step" or "stage".

**Comment by vweevers** [comment]
@ralphtheninja because the build failed, again with that `plan != count` failure from https://github.com/airtap/airtap/issues/152#issuecomment-392011017, I made a fix that we can squeeze into 0.0.9. PR coming up.

**Comment by ralphtheninja** [comment]
Rebased and added entry in changelog.

**Pull request merged**